### ISSUE TITLE
Tech Order not Working via Options

### ISSUE THREAD

**Issue by thebeachtoday** [videojs/videojs-contrib-hls#452]
Website: www.johnspass.tv
Functional Issue: HLS streams cannot play when integrated with non-x-application/m3u8 calls via external JavaScript functions
Functional Issue Limitations: iPhone and Android [i.e. devices that do NOT use flash] do NOT have the issue - just Firefox and Chrome on PC.  I have not tested on Safarai on Apple.
Functional Issue Recreation:  Using Chrome or Firefox on PC: Upon visiting www.JohnsPass.TV, play the video [so you can see how it works], click Guide on the remote [tap remote edge if hidden].  On the second row in the channel guide table, click the right arrow.  An option "Streaming Live!" will appear, click the text.  Notice in the upper right the stream does not launch.  Test any other show in the table to see how it should work.
Technical Approach: Disable Flash so that we always use HTML5 player.  Since we have an issue with slow loading [here: https://github.com/videojs/video.js/issues/2341], options are set in a script at the end of the file: player.options.techOrder = ['html5'];
Technical Issue: Cannot disable flash via command above
Other Technical Approaches:  I am open to other solutions, however, to get this to load fast, I like changes the video src tag to quickly play any video [or stream].  This is not possible with the current way that flash overrides all video tags.   Once we lose the video tag, we have an object [i.e. flash] tag.

Bottom Line: I need to quickly switch between videos and live streaming.  It works fine on iPhone and Android - why can't we get it to work on Firefox and Chrome?

PS: I do not care about IE, Windows Phone, Blackberry, etc.  Just need it to work standard web browsers on iPhone 5 or later and new Android phones as well as Safari, Chrome and Firefox on PC/Apple.

**Comment by gesinger** [comment]
Sorry this ticket has gone a long time without much attention, and thanks for the update with your work around. I think this case should work with the latest player. If you still need it without the workaround, please give the latest version (as of this comment v5.2.1) a shot and let us know if you're still seeing issues. Thank you.
